wappalyzer.js

```diff
@@ -269,19 +269,22 @@ const Wappalyzer = {
               throw new Error(`Implied technology does not exist: ${name}`)
             }
 
-            if (
-              resolved.findIndex(
-                ({ technology: { name } }) => name === implied.name
-              ) === -1
-            ) {
+            const existingIndex = resolved.findIndex(
+              ({ technology: { name } }) => name === implied.name
+            )
+            if (existingIndex === -1) {
               resolved.push({
                 technology: implied,
-                confidence: Math.min(confidence, _confidence),
+                confidence: Math.max(confidence, _confidence),
```

Comment:
`_confidence` is de confidence van de implication, niet van de match. Je kan een `implies` ook een confidence geven. Die is bijna altijd 100 (want weggelaten in `"implies"`). Die moet je niet gebruiken om de match te verbeteren.

Reply:
Ah, okay. Helder.